src/index.js

```diff
@@ -156,9 +176,15 @@ const initPlayer = (params, embed) => {
  *         A function to call if a player fails to be initialized.
  */
 const loadPlayer = (params, resolve, reject) => {
+  params.refNode = resolveRefNode(params.refNode);
```

Comment:
This updates the `params` object temporarily. Once it passes through `createEmbed()` this property will be nulled out to avoid potential memory leaks.